components/display/index.rst

```diff
@@ -9,10 +9,10 @@ The ``display`` component houses ESPHome's powerful rendering and display
 engine. Fundamentally, there are these types of displays:
 
 - Text based displays like :doc:`7-Segment displays <max7219>` or
-  :doc:`some LCD displays <lcd_display>`.
-- Displays like the :doc:`nextion` that have their own processors for rendering.
-- Binary displays which can toggle ON/OFF any pixel, like :doc:`E-Paper displays <waveshare_epaper>` or
-  :doc:`OLED displays <ssd1306>`.
+  :doc:`LCD displays <lcd_display>`.
```

Comment:
"Liquid Crystal Display displays"...stuff like this always triggers my OCD. 😅

Reply:
In my defence, that was already there, just got moved a bit :-)

Reply:
Haha yes indeed, I noticed 😅